homeassistant/components/freebox/router.py

```diff
@@ -89,7 +93,39 @@ async def update_all(self, now: datetime | None = None) -> None:
     async def update_device_trackers(self) -> None:
         """Update Freebox devices."""
         new_device = False
-        fbx_devices: list[dict[str, Any]] = await self._api.lan.get_hosts_list()
+
+        fbx_devices: list[dict[str, Any]] = []
+
+        # Access to Host list not available in bridge mode, API return error_code 'nodev'
+        if self.supports_hosts:
+            hosts_list_initialized = False
+            try:
+                fbx_devices.append(await self._api.lan.get_hosts_list())
+                hosts_list_initialized = True
+            except HttpRequestError as err:
+                m = re.search('Request failed \(APIResponse: (.+?)\)', str(err))
```

Comment:
I don't like one letter var, maybe call it "matcher" ?

Reply:
Done